### PR TITLE
infinty loop

### PR DESCRIPTION
**Pull request by manikasharma09**
In this we are creating a loop which goes around like forever when we place the mouse pointer on that.

### CODE REVIEW

**Comment by greybax** [comment]
@manikasharma09 Thanks for your PR! Could you please provide some information what this PR do or add some screenshots?

**Comment by manikasharma09** [comment]
sure

**Comment by manikasharma09** [comment]
written the description of what this code does

**Comment by greybax** [comment]
@manikasharma09 not sure why we need this... Is it will help somehow to navigate on website or what?

File: h.html.txt [hunk 1]
```diff
@@ -0,0 +1,72 @@
+HTML
```
Comment (greybax) [inline h.html.txt]: Sorry but this PR doesn't make sense, because it's not related with theme of this repo and you just add a `.txt` file.

**Pull request closed without merging**